app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectTypesMessages.js

```diff
@@ -57,6 +57,10 @@ export const messages = defineMessages({
     id: 'DefectTypesTab.deleteModalHeader',
     defaultMessage: 'Delete Defect Type',
   },
+  copyToClipboardLocator: {
+    id: 'Common.copyToClipboardLocator',
```

Comment:
Please, rename 'Common.copyToClipboardLocator' to 'DefectTypesTab.copyToClipboardLocator'

Reply:
Done.